main/exercise/question.class.php

```diff
@@ -130,15 +136,15 @@ public function getIsContent()
      *
      * @author Olivier Brouckaert
      */
-    public static function read($id, $course_info = [], $getExerciseList = true)
+    public static function read($id, $course_info = [], $getExerciseList = true, $ptest = false)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The variable $course_id is not named in camelCase.](https://app.codacy.com/manual/chamilo/chamilo-lms/pullRequest?prid=5018478)